README.md

```diff
@@ -1,5 +1,9 @@
 # crashsimilarity
 
+[![GitHub version](https://badge.fury.io/gh/marco-c%2Fcrashsimilarity.svg)](https://badge.fury.io/gh/marco-c%2Fcrashsimilarity)
+
```

Comment:
Can you remove this blank line? This way the badges will be on the same line in the resulting output: https://github.com/aditya-iitd/crashsimilarity/blob/5699c976cfde783d1db68e3dcdfea81c13d0c5ef/README.md.

Actually, let's remove the version badge for now. We will add it in the future if we decide to publish this package on PyPI.